src/Enterspeed.Source.UmbracoCms.V7/Services/EnterspeedJobService.cs

```diff
@@ -0,0 +1,90 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Enterspeed.Source.UmbracoCms.V7.Contexts;
+using Enterspeed.Source.UmbracoCms.V7.Data.Models;
+using Enterspeed.Source.UmbracoCms.V7.Data.Repositories;
+using Enterspeed.Source.UmbracoCms.V7.Extensions;
+using Enterspeed.Source.UmbracoCms.V7.Models.Api;
+using Umbraco.Core;
+using Umbraco.Core.Models;
+using Umbraco.Core.Services;
+using Umbraco.Web.Models;
+
+namespace Enterspeed.Source.UmbracoCms.V7.Services
+{
+    public class EnterspeedJobService
+    {
+        private readonly IContentService _contentService;
+        private readonly EnterspeedJobRepository _enterspeedJobRepository;
+
+        public EnterspeedJobService()
+        {
+            _contentService = ApplicationContext.Current.Services.ContentService;
+            _enterspeedJobRepository = EnterspeedContext.Current.Repositories.JobRepository;
+        }
+
+        public SeedResponse Seed()
+        {
+            var allContent = new List<IContent>();
+
+            // Add all root nodes
+            var rootContent = _contentService.GetRootContent().ToList();
```

Comment:
We are only getting published content here, right?

Reply:
No, actually we got content from db. I've changed so it seeds from the Umbraco cache instead